pkgs/applications/virtualization/nvidia-docker/libnvc.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, libelf, libcap, libseccomp
+, nvidia_x11 }:
+
+with lib; let
+
+  persistenced = nvidia_x11.passthru.persistenced;
+
+in stdenv.mkDerivation rec {
+  name = "libnvidia-container-${version}";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "libnvidia-container";
+    rev = "deccb2801502675bd283c6936861814dbca99ecd";
```

Comment:
and here: https://github.com/NVIDIA/libnvidia-container/releases

Reply:
Done